packages/edit-site/src/components/page-templates/dataviews-templates.js

```diff
@@ -167,10 +168,19 @@ export default function DataviewsTemplates() {
 				getValue: ( { item } ) => item.description,
 				render: ( { item } ) => {
 					return (
-						item.description && (
+						item.description ? (
 							<Text variant="muted">
 								{ decodeEntities( item.description ) }
 							</Text>
+						) : (
+							<>
+							<Text variant="muted" aria-hidden="true">
+								{ __( '—' ) }
```

Comment:
Unsure if this needs to be translatable 🤔

Reply:
It's not a translatable string in core, so I think you can use the same decision here. I suggest using the same character as is used in core, for consistency: `&#8212;`